### PR TITLE
chore: rename some config fields and uncomment tests back

### PR DESCRIPTION
**Pull request by tejal29**
Bring back uncommented tests in config_test.go

Merge after #7418

Rename `Render `to `Manifest`
Keep `RenderConfig` as `RenderConfig`

### CODE REVIEW

**Comment by tejal29** [comment]
Work on this after #7418

File: pkg/skaffold/runner/runcontext/v2/context.go [hunk 1]
```diff
@@ -276,7 +276,7 @@ func (rc *RunContext) JSONParseConfig() latest.JSONParseConfig {
 func (rc *RunContext) GetRenderConfig() *latest.RenderConfig {
```
Comment (gsquared94) [inline pkg/skaffold/runner/runcontext/v2/context.go:276]: We can delete `GetRenderConfig`. It's not used anywhere.

Comment (gsquared94) [inline pkg/skaffold/runner/runcontext/v2/context.go:276]: We can delete `GetRenderConfig`. It's not used anywhere.

**Review by gsquared94 — APPROVED** [review]

File: cmd/skaffold/app/cmd/run.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ func doRun(ctx context.Context, out io.Writer) error {
 			}
 		}
 
-		// Render
+		// Manifests
```
Comment (MarlonGamez) [inline cmd/skaffold/app/cmd/run.go:57]: this should probably be kept the same or just removed as the comment doesn't add anything really

Comment (aaron-prindle) [reply cmd/skaffold/app/cmd/run.go:57]: done (in next version of PR)

File: pkg/skaffold/runner/runcontext/v2/context.go (hunk 1 above)
Comment (aaron-prindle) [reply pkg/skaffold/runner/runcontext/v2/context.go:276]: done (in next version of PR)

**Pull request closed without merging**